cpp/src/groupby/sort/aggregate.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include <groupby/common/utils.hpp>
+#include "functors.hpp"
```

Comment:
Although these files may be at the same folder as the current file, I would prefer to use the full path with brackets (`<cudf/groupby/....hpp`).